Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                string name = PromptString("Name: ");
```

Comment:
The body of for loop could be a method as well- ProcessPersonBmi or something like that.